tcms/static/js/bugs.js

```diff
@@ -49,3 +47,21 @@ function loadBugs(selector, filter) {
                 bugDetailsCache);
         });
 }
+
+function fetchBugDetails(source, popover, cache = bugDetailsCache) {
+    if (source.href in cache) {
+        assignPopoverData(source, popover, cache[source.href]);
+        return;
+    }
+
+    jsonRPC('Bug.details', [source.href], data => {
+        cache[source.href] = data;
+        assignPopoverData(source, popover, data);
+    }, true);
```

Comment:
@atodorov is there a valid reason for this to originally be `true`. we do nothing after this call, so it could be async. that way the browser won't complain about blocking tasks in the main thread.

Reply:
Probably because the original usage was assigned to an on-show event and when this is executed async it finishes after the popover is shown.